crates/kornia-tensor-ops/src/error.rs

```diff
@@ -11,4 +12,12 @@ pub enum TensorOpsError {
     /// Tensor error
     #[error("Error with the tensor: {0}")]
     TensorError(#[from] TensorError),
+
+    /// Tensor error
+    #[error("Error with the kernel: {0}")]
```

Comment:
```suggestion
    #[transparent]
```
I recently learnt this trick, to forward directly the error